[2024-07-03T22:06:24.176+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-03T22:06:24.233+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: exchange_rate_dag.run_script scheduled__2024-03-07T00:00:00+00:00 [queued]>
[2024-07-03T22:06:24.247+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: exchange_rate_dag.run_script scheduled__2024-03-07T00:00:00+00:00 [queued]>
[2024-07-03T22:06:24.266+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-03T22:06:24.409+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_script> on 2024-03-07 00:00:00+00:00
[2024-07-03T22:06:24.432+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_dag', 'run_script', 'scheduled__2024-03-07T00:00:00+00:00', '--job-id', '112', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmppi59w8vn']
[2024-07-03T22:06:24.431+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=182) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-03T22:06:24.451+0000] {standard_task_runner.py:91} INFO - Job 112: Subtask run_script
[2024-07-03T22:06:24.476+0000] {standard_task_runner.py:63} INFO - Started process 191 to run task
[2024-07-03T22:06:24.674+0000] {task_command.py:426} INFO - Running <TaskInstance: exchange_rate_dag.run_script scheduled__2024-03-07T00:00:00+00:00 [running]> on host bab0a72648e8
[2024-07-03T22:06:26.053+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='juan_ml' AIRFLOW_CTX_DAG_ID='exchange_rate_dag' AIRFLOW_CTX_TASK_ID='run_script' AIRFLOW_CTX_EXECUTION_DATE='2024-03-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-07T00:00:00+00:00'
[2024-07-03T22:06:26.057+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-03T22:06:26.059+0000] {logging_mixin.py:188} INFO - All credentials are set properly.
[2024-07-03T22:06:26.060+0000] {logging_mixin.py:188} INFO - Error al crear la tabla en Redshift: invalid literal for int() with base 10: 'None'
[2024-07-03T22:06:27.444+0000] {logging_mixin.py:188} INFO - Error al realizar la solicitud a la API: 401 Client Error: Unauthorized for url: https://api.apilayer.com/exchangerates_data/latest?symbols=EUR,USD,CNY&base=ARS
[2024-07-03T22:06:27.447+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-03T22:06:27.449+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-03T22:06:27.464+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=exchange_rate_dag, task_id=run_script, run_id=scheduled__2024-03-07T00:00:00+00:00, execution_date=20240307T000000, start_date=20240703T220624, end_date=20240703T220627
[2024-07-03T22:06:27.643+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-03T22:06:27.671+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T22:06:27.742+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-03T23:03:54.330+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-03T23:03:54.946+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: exchange_rate_dag.run_script scheduled__2024-03-07T00:00:00+00:00 [queued]>
[2024-07-03T23:03:55.630+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: exchange_rate_dag.run_script scheduled__2024-03-07T00:00:00+00:00 [queued]>
[2024-07-03T23:03:55.636+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-03T23:03:56.049+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_script> on 2024-03-07 00:00:00+00:00
[2024-07-03T23:03:56.594+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_dag', 'run_script', 'scheduled__2024-03-07T00:00:00+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpfoct1zh2']
[2024-07-03T23:03:57.177+0000] {standard_task_runner.py:91} INFO - Job 43: Subtask run_script
[2024-07-03T23:03:56.456+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=293) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-03T23:03:57.582+0000] {standard_task_runner.py:63} INFO - Started process 320 to run task
[2024-07-03T23:03:59.258+0000] {task_command.py:426} INFO - Running <TaskInstance: exchange_rate_dag.run_script scheduled__2024-03-07T00:00:00+00:00 [running]> on host 0b7358cba041
[2024-07-03T23:04:02.622+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='juan_ml' AIRFLOW_CTX_DAG_ID='exchange_rate_dag' AIRFLOW_CTX_TASK_ID='run_script' AIRFLOW_CTX_EXECUTION_DATE='2024-03-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-07T00:00:00+00:00'
[2024-07-03T23:04:03.007+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-03T23:04:03.024+0000] {logging_mixin.py:188} INFO - All credentials are set properly.
[2024-07-03T23:04:03.072+0000] {logging_mixin.py:188} INFO - Error al crear la tabla en Redshift: invalid literal for int() with base 10: 'None'
[2024-07-03T23:04:06.885+0000] {logging_mixin.py:188} INFO - Error al realizar la solicitud a la API: 401 Client Error: Unauthorized for url: https://api.apilayer.com/exchangerates_data/latest?symbols=EUR,USD,CNY&base=ARS
[2024-07-03T23:04:06.933+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-03T23:04:06.986+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-03T23:04:07.389+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=exchange_rate_dag, task_id=run_script, run_id=scheduled__2024-03-07T00:00:00+00:00, execution_date=20240307T000000, start_date=20240703T230354, end_date=20240703T230407
[2024-07-03T23:04:07.766+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-03T23:04:07.907+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T23:04:07.947+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-04T01:39:35.273+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-04T01:39:35.307+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: exchange_rate_dag.run_script scheduled__2024-03-07T00:00:00+00:00 [queued]>
[2024-07-04T01:39:35.319+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: exchange_rate_dag.run_script scheduled__2024-03-07T00:00:00+00:00 [queued]>
[2024-07-04T01:39:35.320+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-04T01:39:35.340+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_script> on 2024-03-07 00:00:00+00:00
[2024-07-04T01:39:35.353+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=419) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-04T01:39:35.356+0000] {standard_task_runner.py:63} INFO - Started process 421 to run task
[2024-07-04T01:39:35.356+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_dag', 'run_script', 'scheduled__2024-03-07T00:00:00+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmp3poggwd3']
[2024-07-04T01:39:35.358+0000] {standard_task_runner.py:91} INFO - Job 37: Subtask run_script
[2024-07-04T01:39:35.434+0000] {task_command.py:426} INFO - Running <TaskInstance: exchange_rate_dag.run_script scheduled__2024-03-07T00:00:00+00:00 [running]> on host e26a552f405a
[2024-07-04T01:39:35.602+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='juan_ml' AIRFLOW_CTX_DAG_ID='exchange_rate_dag' AIRFLOW_CTX_TASK_ID='run_script' AIRFLOW_CTX_EXECUTION_DATE='2024-03-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-07T00:00:00+00:00'
[2024-07-04T01:39:35.605+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-04T01:39:35.606+0000] {logging_mixin.py:188} INFO - All credentials are set properly.
[2024-07-04T01:39:35.607+0000] {logging_mixin.py:188} INFO - Error al crear la tabla en Redshift: invalid literal for int() with base 10: 'None'
[2024-07-04T01:39:36.084+0000] {logging_mixin.py:188} INFO - Error al realizar la solicitud a la API: 401 Client Error: Unauthorized for url: https://api.apilayer.com/exchangerates_data/latest?symbols=EUR,USD,CNY&base=ARS
[2024-07-04T01:39:36.087+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-04T01:39:36.089+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-04T01:39:36.103+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=exchange_rate_dag, task_id=run_script, run_id=scheduled__2024-03-07T00:00:00+00:00, execution_date=20240307T000000, start_date=20240704T013935, end_date=20240704T013936
[2024-07-04T01:39:36.175+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-04T01:39:36.198+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-04T01:39:36.202+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
